packages/dx-react-grid/docs/guides/plugin-overview.md

```diff
@@ -1,6 +1,22 @@
 # React Grid - Plugin Overview
 
-The plugins that provide all the Grid's functionality can be divided into four groups:
+Plugins are components which should be placed inside Grid and responsible for enabling particular Grid features. The core principle used for supplying plugins with data is piping. This means that data flows consequently from the first plugin to the next one and so on. And this is where [plugin order](#pluginorder) comes to a scene.
```

Comment:
I think we can use this link [piping](https://en.wikipedia.org/wiki/Pipeline_(computing))